README.md

```diff
@@ -108,25 +122,25 @@ $ pigz -p 10 --stdout > output1.fastq.gz < output1.fastq &
 $ pigz -p 10 --stdout > output2.fastq.gz < output2.fastq &
 [5] 233395
 ```
+
 The value `10` is how many threads each of the `pigz` processes is allowed to use.
 The optimal value for this depends on several factors and for optimal performance you will have to do some testing on your exact hardware.
 We can then run the `umi-transfer` program as follows:
+
 ```shell
-$ umi-transfer --prefix output --edit-nr --r1-in read1.fastq --r2-in read3.fastq --ru-in read2.fastq
+umi-transfer --in read1.fastq --in2 read3.fastq --umi read2.fastq --out output1.fastq --out2 output2.fastq
```

Comment:
`--force` is never needed, but I would recommend using it for non-interactive use. But I forgot an `external` here, so great bringing this line to my attention ;-)

Reply:
True, I guess I have my mindset on non-interactive mode. I would still include a `--force` in this particular line though since users might get confused about the question if they're not accustomed to use fifos. What do you think?

Reply:
Finally, the penny has dropped. I agree, we should put it in there.